### PR TITLE
Feature/may young detail 1

### PR DESCRIPTION
**Pull request by MayHyeyeonKim**
Hey team,
Ive added a` Daum postal-code API `to the prev detail page task 1.
Plz take a look and let me know your feedback! :)

Screenshot 2024-09-05 at 12 04 01 PM

### CODE REVIEW

File: src/utils/AddressChange.js
Comment (hunjoonrhee) [inline src/utils/AddressChange.js:1]: is this a react component? if so, why did you create it as a .js file?

File: src/utils/DeliveryEstimate.js
Comment (hunjoonrhee) [inline src/utils/DeliveryEstimate.js:1]: same question as for AddressChange.js

**Comment by hunjoonrhee** [comment]
In general, it looks good, only I think that the two .js files that serve as react components should be converted to tsx format.

**Comment by MayHyeyeonKim** [comment]
> In general, it looks good, only I think that the two .js files that serve as react components should be converted to tsx format.

If you're referring to the files in `utils`, should I also convert `helpers.js`, `AddressChange.js`, and `DeliveryEstimate.js` to `tsx` while 바꾸는김에?

**Comment by hunjoonrhee** [comment]
good!

**Review by hunjoonrhee — APPROVED** [review]

**Pull request merged**